march_shared_classes/src/march_shared_classes/gait/subgait.py

```diff
@@ -1,13 +1,19 @@
+import os
+import re
+
 import rospy
 from trajectory_msgs import msg as trajectory_msg
 import yaml
 
 from march_shared_classes.exceptions.gait_exceptions import NonValidGaitContent, SubgaitInterpolationError
+from march_shared_classes.exceptions.general_exceptions import FileNotFoundError
 
 from .joint_trajectory import JointTrajectory
 from .limits import Limits
 from .setpoint import Setpoint
 
+PARAMETRIC_GAITS_CHARACTER = '_'
```

Comment:
It might be nice to name it a prefix to not limit it to one character
```suggestion
PARAMETRIC_GAITS_PREFIX = '_'
```